.circleci/config.yml

```diff
@@ -22,19 +22,19 @@ jobs:
           key: yarn-packages-{{ checksum "yarn.lock" }}
           paths:
             - ~/.cache/yarn
-      - run: 
+      - run:
           name: Build
           command: yarn run build
-      - run: 
-          name: Lint
-          command: yarn run lint
```

Comment:
why lint gone??

Reply:
no need to have linting if we already have prettier to format the code.